src/main/java/org/junitpioneer/jupiter/CartesianValueSource.java

```diff
@@ -99,9 +99,15 @@
 	 */
 	Class<?>[] classes() default {};
 
+	/**
+	 * Containing annotation of repeatable {@code CartesianValueSource}
```

Comment:
Below we end the sentence with a period (.):

```suggestion
	 * Containing annotation of repeatable {@code CartesianValueSource}.
```